configs/coco8.yaml

```
@@ -0,0 +1,101 @@
+  71: sink
+  72: refrigerator
+  73: book
+  74: clock
+  75: vase
+  76: scissors
+  77: teddy bear
+  78: hair drier
+  79: toothbrush
+
+# Download script/URL (optional)
+download: https://github.com/ultralytics/assets/releases/download/v0.0.0/coco8.zip
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)